pkg/auth/providers/github/github_account.go

```diff
@@ -26,6 +29,12 @@ type Team struct {
 	Slug         string                 `json:"slug,omitempty"`
 }
 
+// EmailResponse is used for getting response from the emails endpoint. https://developer.github.com/v3/users/emails/#response .We only need the Email and Primary fields
+type EmailResponse struct {
+	Email   string `json:"email,omitempty"`
+	Primary bool   `json:"primary,omitempty"`
+}
```

Comment:
Does this also list user's private email addresses?

Reply:
yes, but only after user approves the user:email scope we request during login

Reply:
Will user be approving it on the configuration page for github? What if user does not approve it

Reply:
User will approve it during login. If user doesn't click "authorize" then login won't take place.  It's similar to the read:org access we already request

Reply:
ok